lib/enumerations/value.rb

```diff
@@ -16,6 +16,10 @@ def to_sym
       symbol
     end
 
+    def quoted_id
```

Comment:
@d4be4st without `quoted_id`, with_enumeration(enumeration) scope will not work if we send Enumerations.value object:

`User.with_role(Role.admin, 'editor')`